src/test/java/com/redhat/devtools/lsp4ij/fixtures/LSPClientSideOnTypeFormattingFixtureTestCase.java

```diff
@@ -49,10 +49,21 @@ public abstract class LSPClientSideOnTypeFormattingFixtureTestCase extends LSPCo
 
     private static final Pattern TYPE_INFO_PATTERN = Pattern.compile("(?ms)//\\s*type\\s*(\\S)[\\t ]*");
 
+    private boolean supportsRangeFormatting = true;
```

Comment:
Derived test classes can specify that the tested language server doesn't support range formatting if appropriate.